tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteOffsetsConsumerGroupCommandIntegrationTest.java

```diff
@@ -213,4 +246,9 @@ private Consumer<byte[], byte[]> createConsumer(Properties config) {
 
         return new KafkaConsumer<>(config);
     }
+
+    private void createTopic(String topic) {
+        Assertions.assertDoesNotThrow(() ->
+            clusterInstance.createAdminClient().createTopics(Collections.singletonList(new NewTopic(topic, 1, (short) 1))).topicId(topic).get());
```

Comment:
Could we use try-with-resource to close `Admin`? We don't guarantee that `createAdminClient` will auto-release the admin currently. I have filed https://issues.apache.org/jira/browse/KAFKA-16589 to discuss it. However, we should always close it.